tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_live_os.py

```diff
@@ -206,17 +230,23 @@ def test_setup_install_source_task_run(self, proxy_getter, os_stat, stat, mount)
 
         mount.return_value = 0
 
-        SetUpLiveOSSourceTask(
+        result = SetUpLiveOSSourceTask(
             "/path/to/base/image",
             "/path/to/mount/source/image"
         ).run()
 
+        assert isinstance(result, SetupLiveOSResult)
+        assert result.required_space == 12345
+
         device_tree.ResolveDevice.assert_called_once_with("/path/to/base/image")
         os_stat.assert_called_once_with("/resolved/path/to/base/image")
 
+    @patch("pyanaconda.modules.payloads.source.live_os.initialization.SetUpLiveOSSourceTask"
+           "._calculate_required_space")
```

Comment:
Please, replace this patch in other tests as well.

Reply:
I think I got them all this time, sorry about that.